graphql/schema/types/scene.graphql

```diff
@@ -52,6 +52,10 @@ type Scene {
   created_at: Time!
   updated_at: Time!
   file_mod_time: Time
+  last_played_at: Time
+  resume_time: Float
+  play_duration: Float
+  play_count: Int
```

Comment:
Please document these fields.

Reply:
By document are you asking for comments about these fields?